packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -450,11 +452,14 @@ class VotesRepo extends AbstractRepo<"Votes"> {
         AND ($2 IS NULL OR v."createdAt" >= $2)
         AND v."createdAt" <= $3
         AND v."cancelled" IS NOT TRUE
+        AND v."isUnvote" IS NOT TRUE
+        AND v."voteType" != 'neutral'
+        AND NOT "authorIds" @> ARRAY["userId"]
```

Comment:
I added these filters here, and I think this query is used on the Author Analytics page as well? I think this is right but let me know if something seems off